eng/Versions.props

```diff
@@ -135,7 +135,7 @@
     <AzureSearchDocumentsVersion>11.6.0</AzureSearchDocumentsVersion>
     <MicrosoftSemanticKernelConnectorsAzureAISearchVersion>1.37.0-preview</MicrosoftSemanticKernelConnectorsAzureAISearchVersion>
     <MicrosoftSemanticKernelCoreVersion>1.37.0</MicrosoftSemanticKernelCoreVersion>
-    <OllamaSharpVersion>5.0.7</OllamaSharpVersion>
+    <OllamaSharpVersion>5.1.5</OllamaSharpVersion>
```

Comment:
Please note this will result in a conflict after https://github.com/dotnet/extensions/pull/6108